src/patternfly/components/AboutModalBox/about-modal-box.scss

```diff
@@ -2,31 +2,9 @@
   // Component variables
   --pf-c-about-modal-box--BackgroundColor: var(--pf-global--palette--black-1000); // Modal uses a non-standard background color
   --pf-c-about-modal-box--BoxShadow: 0 0 100px 0 rgba(255, 255, 255, .05);  // This is a special one-off glow for the about modal
-  --pf-c-about-modal-box--ZIndex: var(--pf-global--ZIndex--xl);
-  --pf-c-about-modal-box--Height: 100%;
-  --pf-c-about-modal-box--lg--Height: #{pf-size-prem(762px)}; // Height is optimized for the exact height desired
-  --pf-c-about-modal-box--lg--MaxHeight: calc(100% - var(--pf-global--spacer--xl)); // MaxHeight ensures that the modal will not go off the screen and instead the content will scroll
```

Comment:
Hmmm... removing these heights is going to make the about modal shrink to fit its contents. Maybe that's a good thing? We had this use case when we moved the wizard into the modal component, too, and we ended up giving it a height when it's inside of a modal. https://github.com/patternfly/patternfly/blob/7857f7ed377cc40bdf7721a1c9b710b7f34258a0/src/patternfly/components/Wizard/wizard.scss#L207-L211

This is what the about modal looks like now on org with minimal content:

<img width="1945" alt="Screen Shot 2022-11-07 at 5 21 24 PM" src="https://user-images.githubusercontent.com/35148959/200436348-6a951c0f-35a3-4ad9-ac8e-e4fb6c5f5b2d.png">

And this is what it looks like in this PR:

<img width="1945" alt="Screen Shot 2022-11-07 at 5 21 52 PM" src="https://user-images.githubusercontent.com/35148959/200436405-2e603cd7-75bb-4efe-a04f-6992ced3c8cd.png">

@mcarrano @mceledonia @mmenestr wdyt?

Reply:
I prefer having the negative space even with limited content, it provides a nice balance which gets skewed when the height is set to fit.

Reply:
Agree

Reply:
@mceledonia @mcarrano will this component always be inside of a modal, or do you see a use case where it may be outside of a modal - either now or in the future with some other kind of component that can present it?

Reply:
I suppose you could have an "About page," but I don't really see a need for this. A simple modal should suffice in most cases. @mceledonia @mmenestr do you agree?

Reply:
Height retained